neo/test/rawiotest/test_maxwellrawio.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+import os
+
+from neo.rawio.maxwellrawio import (MaxwellRawIO,
+    auto_install_maxwell_hdf5_compression_plugin)
+from neo.test.rawiotest.common_rawio_test import BaseTestRawIO
+
+in_gh_actions = os.getenv('GITHUB_ACTIONS', 'False') == 'true'
+
+
+@unittest.skipUnless(not in_gh_actions, "Need specific hdf5 plugin")
```

Comment:
Does this mean that the test might fail locally even when the user did install all official requirements? Shouldn't this test rather be only executed if hdf5 is present in the first place?

Reply:
Maybe we should remove this from test always.
The test is working on my local machine.

Reply:
Ah, earlier I didn't realize you are talking about the compression library here. I think it would be best to check if the plugin is installed and only run the test if this is the case.

So the auto installation does not work on github actions?

Reply:
No it was working, this is why I disable.
I need to check again.

Reply:
@samuelgarcia did you check again?

Reply:
This line is outdated. I remove the skip and now the maxwell is tested and test are passing.